### ISSUE TITLE
`sortBy` doesn't sort the query result

### ISSUE THREAD

**Issue by smarroufin** [nuxt/content#1064]
### Environment

- Operating System: `Darwin`
- Node Version:     `v14.17.0`
- Nuxt Version:     `3.0.0-rc.3`
- Package Manager:  `yarn@1.22.10`
- Builder:          `vite`
- User Config:      `css`, `modules`, `build`, `runtimeConfig`, `strapi`, `ui`, `tailwindcss`, `content`, `typescript`
- Runtime Modules:  `@nuxthq/ui@0.0.2-27536218.7275668`, `@nuxthq/admin@0.0.1-27531272.b4f169e`, `@nuxtjs/strapi@1.3.2-27529127.14eaac5`, `@nuxt/content@2.0.0-27536649.61e4b1f`, `vue-plausible@1.3.1`
- Build Modules:    `-`

### Reproduction

Let's say we have blog posts with a date in the front matter:
`1.article-1.md`
```
---
date: 2021-09-10
---
```
`2.article-2.md`
```
---
date: 2022-01-01
---
```

`queryContent('/resources/blog').where({ $not: { slug: { $in: ['/resources/blog'] } } }).sortBy('date', 'desc').find()`

### Describe the bug

The order should be affected by the `sortBy`. But it isn't.

**Comment by pidgpowell** [comment]
@farnabaz thank you! i can now sort!!

**Comment by BroJenuel** [comment]
const blogs = await queryContent("blog").where({ active: 1 }).sort(number).find();

**Comment by farnabaz** [comment]
@BroJenuel  Yes, you can use `sort` to order results. Check out [sort docs](https://content.nuxtjs.org/api/composables/query-content#sortoptions)

**Comment by PuxianAlHazred** [comment]
> @farnabaz thank you! i can now sort!!

thx you for ?
